blocks/library/freeform/old-editor.js

```diff
@@ -168,20 +162,20 @@ export default class OldEditor extends Component {
 	}
 
 	render() {
-		const { isSelected, id, className } = this.props;
+		const { isSelected, id } = this.props;
 
 		return [
 			<div
 				key="toolbar"
-				id={ id + '-toolbar' }
+				id={ `toolbar-${ id }` }
 				ref={ ref => this.ref = ref }
 				className="freeform-toolbar"
 				style={ ! isSelected ? { display: 'none' } : {} }
 			/>,
 			<div
 				key="editor"
-				id={ id }
-				className={ classnames( className, 'blocks-rich-text__tinymce' ) }
+				id={ `editor-${ id }` }
+				className="wp-block-freeform blocks-rich-text__tinymce"
```

Comment:
restored the `wp-block-freeform` className to fix the editor stylesheets.  This has broken in a recent PR removing className support